### ISSUE TITLE
Update tools to correctly rename internal references correctly

### ISSUE THREAD

**Issue by jabrah** [jhu-digital-manuscripts/rosa2#107]
Must rename references to follow archive naming scheme.

Lower priority right now, but all data must be updated once this is done (currently, nothing is done with internal references).

**Comment by jabrah** [comment]
Done:
 * c15d75d086c14f2a7f50686d556c88f11b443652
 * 45e1baf3e1637551056eb9f5d87d27a95abb47a3
